spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/KubernetesModuleDeployerProperties.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.module.deployer.kubernetes;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Florian Rosenberg
+ */
+@ConfigurationProperties("kubernetes")
+class KubernetesModuleDeployerProperties {
+
+	/**
+	 * Use the default spring image. Override with --kubernetes.moduleLauncherImage
+	 * as a Spring Admin parameter to use a different one.
+	 */
+	private static final String DEFAULT_IMAGE_NAME = "springcloud/stream-module-launcher";
+
+	/**
+	 * The Docker image containing the Spring Cloud Stream module launcher.
+	 */
+	private String moduleLauncherImage;
+
+	/**
+	 * Secrets for a access a private registry to pull images.
+	 */
+	private String imagePullSecret;
+
+	/**
+	 * Delay in seconds when the Kubernetes liveness check of the stream module container 
+	 * should start checking its health status.
+	 * 
+	 * See here for more information: 
```

Comment:
All those properties will get documentation metadata information generated thanks to http://docs.spring.io/spring-boot/docs/1.3.0.RC1/reference/htmlsingle/#configuration-metadata

The javadoc should thus be kept to a "minimum", avoiding special tags, etc.
Additional information can be added as a regular (non javadoc) comment